lib/tasks/admin.rake

```diff
@@ -0,0 +1,15 @@
+namespace :admin do
```

Comment:
I actually tried to make it in the Autolab namespace at first and it didn't work. I think it was because there's a lot of hacky stuff going on in it to get auto populate to work. 

Now though, I agree with Dan that it sounds more descriptive.